.github/workflows/artifacts.yml

```diff
@@ -5,7 +5,9 @@ on:
     branches:
       - master
     paths:
-      - 'library/**'
+      - 'library/common/**'
+      - 'library/java/**'
+      - 'library/kotlin/**'
       - 'envoy/**'
 
 jobs:
```

Comment:
we also need to change the job to not compile x86_64. It seems that we have problems with that in CI